airbyte-api/src/main/openapi/config.yaml

```diff
@@ -296,6 +296,11 @@ paths:
         - source_definition
       summary: List all the sourceDefinitions the current Airbyte deployment is configured to use
```

Comment:
Since we decided not to modify the params of the existing endpoints, when implementing the new endpoint behaviors I will also update the summaries of the existing actor definition endpoints to reflect their new behavior.